lib/Doctrine/ORM/Event/LoadClassMetadataEventArgs.php

```diff
@@ -30,12 +32,42 @@
 class LoadClassMetadataEventArgs extends BaseLoadClassMetadataEventArgs
 {
     /**
+     * @param ClassMetadata $classMetadata
+     * @param EntityManager $entityManager
+     */
+    function __construct(ClassMetadata $classMetadata, EntityManager $entityManager)
```

Comment:
Can we at least deprecate it somehow? Nobody should ever be passing in an `ObjectManager` as the second argument. It wasn't even possible in the initial design, until 8443eee6289314813dae87eb7ca92a3a8811ed31.